db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js

```diff
@@ -689,6 +689,15 @@ t3.data->>'CI' as "NBR CELL ID"
 FROM huawei_cm."G2GNCELL" t1
 INNER JOIN huawei_cm."GCELL" t2 on t1.data->>'SRC2GNCELLID'=t2.data->>'CELLID'
 INNER JOIN huawei_cm."GEXT2GCELL" t3 on t1.data->>'NBR2GNCELLID'=t3.data->>'EXT2GCELLID'
+--ZTE 2G-2G RELATIONS
+SELECT 
+'ZTE' as "SRV VENDOR",
+REGEXP_REPLACE(t2.data->>'refGLocationArea','\d+,\d+,(\d+),\d+','\1') AS "SRV LAC",
+t2.data->>'cellIdentity' as "SRV CELL ID",
+REGEXP_REPLACE(t1.data->>'RELATIONCGI','\d+,\d+,(\d+),\d+','\1') AS "NBR LAC",
+REGEXP_REPLACE(t1.data->>'RELATIONCGI','\d+,\d+,\d+,(\d+)','\1') AS "NBR CI"
+FROM zte_cm."GsmRelation" t1
+INNER JOIN ZTE_cm."GsmCell" t2 on t1.data->>'MEID'=t2.data->>'MEID' and t1.data->>'GGsmCellId'=t2.data->>'GGsmCellId' and t1.data->>'GBtsSiteManagerId'=t2.data->>'GBtsSiteManagerId'
```

Comment:
To ensure this is only run for ZTE Plan Template data, add a check for the **dataType** field as it is only available in the Plan Template Excel data. See "WHERE" clause to add below:
`WHERE t1.data->>'dataType' IS NOT NULL AND t2.data->'dataType' IS NOT NULL`

Reply:
included in join